src/components/InstanceEditor.svelte

```diff
@@ -43,6 +64,16 @@
 	<div class="flex flex-row">
 		<div class="flex flex-col justify-center items-center">
 			<button
+				on:drop={handleDrop}
+				on:dragstart={() => {
+					isDragging = true;
```

Comment:
Same here, but also, the `ondragstart` event is fired when the element its attached to starts to be dragged, i.e. when someone starts dragging this button element with the current image, which isn't possible because it isn't dragging. I think you can just remove this block.